AY2021S2-CS2103-T14-1/tp
# Add command history tests

## What this does
This PR:
1. Adds unit tests for various command history components
2. Adds unit tests for `MathUtil`

## How to test
1. Run all tests
2. Ensure they are passing
